frontend/src/pages/Player/context/PlayerUIContext.ts

```diff
@@ -30,10 +30,6 @@ interface PlayerUIContext {
 	setIsPlayerFullscreen: React.Dispatch<React.SetStateAction<boolean>>
 	playerCenterPanelRef: React.RefObject<HTMLDivElement>
 	/** Used to show detailed information. */
-	detailedPanel?: DetailedPanel
-	setDetailedPanel: React.Dispatch<
```

Comment:
Can the `DetailedPanel` and `DetailedPanelOptions` interfaces be cleaned up now as well?